submissions/AnthonyShostak/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,54 @@
+                        </label>
+                        <div class="popup">
+                            <ul class="popup__items">
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/cart.png" alt="cart" class="icon">Cart</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/chat.png" alt="chat" class="icon">Chat</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/favorites.png" alt="favourites" class="icon">Favourites</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/home.png" alt="home" class="icon">Home</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/games.png" alt="games" class="icon">Games</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/location.png" alt="location" class="icon">Location</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/mic.png" alt="microphone" class="icon">Microphone</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/night-mode.png" alt="night-mode" class="icon">Night mode</a></li>
+                                <li class="popup__item"><a href="#" class="link"><img src="popup-icons/scan.png" alt="scan" class="icon">Scan</a></li>
+                                <li class="more__below">
+                                    <input type="checkbox" id="popup__more" class="checkbox__more">
+                                    <label for="popup__more" class="more">Press to see more</label>
+                                    <label for="popup__more" class="less">Press to see less</label>
+                                    <div class="more__inner">
+                                        <ul class="more__inner__content">
+                                            <li class="popup__item"><a href="#" class="link"><img src="popup-icons/trash.png" alt="thrash" class="icon">Thrash</a></li>
+                                            <li class="popup__item"><a href="#" class="link"><img src="popup-icons/settings.png" alt="settings" class="icon">Settings</a></li>
+                                            <li class="popup__item"><a href="#" class="link"><img src="popup-icons/user.png" alt="user" class="icon">User</a></li>
+                                        </ul>
+                                    </div>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                </ul>           
+            </nav>
+        </header>
+        <main class="main">
+        <footer></footer>
+        </main>
+    </body>
+
+    
```

Comment:
Put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.

Reply:
Put empty lines by using "Insert Final Newline" parameter in VCS.